.github/pull_request_template.md

```diff
@@ -2,3 +2,8 @@
 Thanks for opening a PR to scanpy!
 Please be sure to follow the guidelines in our contribution guide (https://scanpy.readthedocs.io/en/latest/dev/index.html) to familiarize yourself with our workflow and speed up review.
 -->
+
+<!-- Please check (“- [x]”) and fill in the following box -->
+- [ ] Closes #
+<!-- Only check the following box if you did not include release nodes -->
+- [ ] Release notes not necessary because:
```

Comment:
```suggestion
<!-- Please check (“- [x]”) and fill in the following boxes -->
- [ ] Closes #
- [ ] Tests included or not required because:
<!-- Only check the following box if you did not include release nodes -->
- [ ] Release notes not necessary because:
```